applications/FluidDynamicsBiomedicalApplication/python_scripts/apply_windkessel_outlet_process.py

```diff
@@ -0,0 +1,138 @@
+        pres_settings.RemoveValue("resistance1")
+        pres_settings.RemoveValue("resistance2")
+        pres_settings.RemoveValue("compliance")
+        pres_settings.RemoveValue("initial_pressure")
+        pres_settings.RemoveValue("venous_pressure")
+        pres_settings.RemoveValue("echo_level")
+
+        # Create a copy of the PRESSURE settings to set the EXTERNAL_PRESSURE
+        ext_pres_settings = pres_settings.Clone()
+        ext_pres_settings["constrained"].SetBool(False)
+        ext_pres_settings["variable_name"].SetString("EXTERNAL_PRESSURE")
+
+        # Check the core processes input data
+        if (pres_settings["model_part_name"].GetString() == ""):
+            raise Exception("Empty outlet pressure model part name. Set a valid model part name.")
+        elif (ext_pres_settings["model_part_name"].GetString() == ""):
+            raise Exception("Empty outlet external pressure model part name. Set a valid model part name.")
+        elif (pres_settings["variable_name"].GetString() != "PRESSURE"):
+            raise Exception("Outlet pressure settings variable_name is not PRESSURE.")
+        elif (ext_pres_settings["variable_name"].GetString() != "EXTERNAL_PRESSURE"):
+            raise Exception("Outlet external pressure settings variable_name is not EXTERNAL_PRESSURE.")
+        elif (pres_settings["value"].IsString()):
+            if (pres_settings["value"].GetString == ""):
+                raise Exception("Outlet pressure function sting is empty.")
+        elif (ext_pres_settings["value"].IsString()):
+            if (ext_pres_settings["value"].GetString == ""):
+                raise Exception("Outlet external pressure function sting is empty.")
+
+        self.R1      = settings["resistance1"].GetDouble()
+        self.R2      = settings["resistance2"].GetDouble()
+        self.C       = settings["compliance"].GetDouble()
+        p0_mmHg      = settings["initial_pressure"].GetDouble()
+        pv_mmHg      = settings["venous_pressure"].GetDouble()
+        self.echo    = settings["echo_level"].GetInt()
+        self.pressure_in_mmHg = settings["pressure_in_mmHg"].GetBool()
+
+        self.conv = 13.545*9.81 # Pressure conversion factor. It is used if pressure is provided in mmHg
+        if self.pressure_in_mmHg is True:
+            self.pv = pv_mmHg*self.conv
+            p0 = p0_mmHg*self.conv
+        else:
+            self.pv = pv_mmHg
+            p0      = p0_mmHg   # The pressure variable passed from the json file is given in Pa
+
+
+        self.previous_q1 = 0.0
+        self.current_p1 = p0 # in Pa
+
+        # Set the OUTLET flag in the outlet model part nodes and conditions
+        self.outlet_model_part = Model[pres_settings["model_part_name"].GetString()]
+        for node in self.outlet_model_part.Nodes:
+            node.Set(KratosMultiphysics.OUTLET, True)
+        for condition in self.outlet_model_part.Conditions:
+            condition.Set(KratosMultiphysics.OUTLET, True)
```

Comment:
You can use the `SetFlag` method from the `VariableUtils` (it is exported to Python) so these two loops are done in parallel.